nuxeo-core/nuxeo-core-test/src/main/java/org/nuxeo/ecm/core/test/CoreFeature.java

```diff
@@ -104,8 +105,8 @@
 @Deploy("org.nuxeo.ecm.platform.commandline.executor")
 @Deploy("org.nuxeo.ecm.platform.el")
 @RepositoryConfig(cleanup = Granularity.METHOD)
-@Features({ RuntimeFeature.class, TransactionalFeature.class, RuntimeStreamFeature.class, WorkManagerFeature.class,
-        CoreBulkFeature.class })
+@Features({ RuntimeFeature.class, TransactionalFeature.class, RuntimeStreamFeature.class, DummyLoginFeature.class,
+        WorkManagerFeature.class, CoreBulkFeature.class })
```

Comment:
Maybe you could update the `CoreFeature`'s Javadoc to explain that the injected session is opened for the principal logged in by the `DummyLoginFeature`: the one passed to `@WithUser`, by default "Administrator" .